.github/workflows/testground.yml

```diff
@@ -0,0 +1,53 @@
+name: Run Testground
+
+on:
+  push:
```

Comment:
We use `push` instead of `pull request` because pull request events [use a different value for](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request ) the`github.sha` var.

Reply:
In standup today, we decided we would like to use `pull request` if possible.

Reply:
Updated in https://github.com/statechannels/go-nitro/pull/891/commits/8725fde15389ace1e164dab7dead432d7ea4738b